index.js

```diff
@@ -26,7 +30,12 @@ class Plugin extends AbstractBtpPlugin {
     this._currencyScale = opts.currencyScale || 9
     this._debugHostIldcpInfo = opts.debugHostIldcpInfo
 
-    this._log = opts._log || console
+    this._logger = opts._log || createLogger
+    this._log = this._logger(DEBUG_NAMESPACE)
```

Comment:
Are you certain that riverpig allows you to do this? The opts._log variable is defined [here](https://github.com/interledgerjs/ilp-connector/blob/master/src/services/accounts.ts#L149) as a fully instantiated riverpig logger. It might be better if your logger class takes the `opts._log` as a parameter and then wraps its  methods if it's defined and otherwise uses the debug module directly.